lib/main/interactions/InteractionManager.ts

```diff
@@ -23,6 +23,11 @@ export class InteractionManager {
     return this.interactionStartTime
   }
 
+  adoptInteractionId(id: string) {
```

Comment:
A bit unrelated to this change but I think it's odd we had different ids for tracing and storing in the db so I merged them